librosa/core/time_frequency.py

```diff
@@ -509,10 +522,10 @@ def note_to_midi(note, round_midi=True):
         return np.array([note_to_midi(n, round_midi=round_midi) for n in note])
 
     pitch_map = {'C': 0, 'D': 2, 'E': 4, 'F': 5, 'G': 7, 'A': 9, 'B': 11}
-    acc_map = {'#': 1, '': 0, 'b': -1, '!': -1}
+    acc_map = {'#': 1, '': 0, 'b': -1, '!': -1, '♯': 1, '𝄪': 2, '♭': -1, '𝄫': -2, '♮': 0}
```

Comment:
I expect naturals to be used exactly never, but may as well support them, right?

OTOH, I'm not sure the logic for accidental accumulation applies correctly here, since nat is (I think?) not equivalent to additive identity.  But I'm not sure if a natural in the middle of a string of accidentals `C♯♮♯` should reset to the left or to the right.  I can't imagine anyone doing this or what they'd expect to happen, so I'm generally okay with leaving it as an additive no-op.

Reply:
Agreed. Let's leave `C♯♮♯` to the obscure corners of https://twitter.com/darkmusictheory